CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## Next
 
+### Added
```

Comment:
Label op issue is `bug` ?

Reply:
Ah, de basis van het issue was ooit een bug. Ik pas het issue aan

Reply:
Met Eric bekeken en goed bevonden